plugins/discovery-ec2/build.gradle

```diff
@@ -46,8 +46,6 @@ dependencies {
   api "commons-logging:commons-logging:${versions.commonslogging}"
   api "org.apache.logging.log4j:log4j-1.2-api:${versions.log4j}"
   api "commons-codec:commons-codec:${versions.commonscodec}"
-  api "com.fasterxml.jackson.core:jackson-databind:${versions.jackson_databind}"
```

Comment:
Why? Is this something we can PR separately?

Let's get all the version bumps for dependencies on main first and reduce the size and goal of this PR to extensions only.

Reply:
In order for extensions to be read in, we needed the jackson dependency to be in server.  This caused errors for "unused" dependencies in the plugins folders, since the jackson dependency was already being read in.

Reply:
I'll raise a PR separately that adds jackson to server in main and we can merge that one first.

Reply:
I can't figure out why the tests are failing for the PR here: https://github.com/opensearch-project/OpenSearch/actions/runs/3550800781/jobs/5964625474

There is no git diff in the discovery-ec2 `build.gradle` file between this branch and that branch but the tests pass on this branch and fail on that one.

Reply:
Not sure, but raise the PR and we can discuss it there.

Reply:
https://github.com/opensearch-project/OpenSearch/pull/5366